apns2/client.py

```diff
@@ -92,9 +92,9 @@ def send_notification_batch(
         if the token was sent successfully, or the string returned by APNs in the 'reason' field of
         the response, if the token generated an error.
         '''
-        token_iterator = iter(tokens)
+        notification_iterator = iter(notifications)
         try:
-            next_token = next(token_iterator)
+            next_notification = next(notification_iterator)
```

Comment:
It would be better because it is more understandable (same as we use for and while loops instead of using the iterator protocol, same as us using + and not __add__()). But I guess we can't circle back to this later.